Formula/xcprojectlint.rb

```diff
@@ -0,0 +1,100 @@
+class Xcprojectlint < Formula
+  desc "Xcode project linter"
+  homepage "https://github.com/americanexpress/xcprojectlint"
+  url "https://github.com/americanexpress/xcprojectlint.git", tag: "0.0.6", revision: "d9dad85847f5ee9b2143565a17d9066bb44b4b29"
+  license "Apache-2.0"
+  head "https://github.com/americanexpress/xcprojectlint.git"
+
+  depends_on xcode: ["12.0", :build]
+
+  def install
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
```

Comment:
Any chance we could simplify this test a bit?

Reply:
Thank you for taking a look at this! 

I managed to shrink the test files a bit. I don't think I can make them any smaller unless I replace the whole thing with something simpler like `xcprojectlint --version`. But I wanted to avoid that based on [the cookbook](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) since it doesn't really test the functionality.

Reply:
Yes, a `--version` test would not be good. Thanks for following the docs.